specify/specify7
# Encode exported Query file paths

Fixes #5069
Fixes #6537

Note: 
The Export process still completely fails if the Query name contains `/` (the character is interpreted as part of the file path). 

### Checklist

      and self-explanatory (or properly documented)

### Testing instructions
- Save and export query with `#` in the query's name
- Export the query as a CSV/KML (without selected any Query Results)
- [ ] Ensure the file can be downloaded and the full query name is in the downloaded file's name
- Save and export query with `?` in the query's name
- Export the query as a CSV/KML (without selected any Query Results)
- [ ] Ensure the file can be downloaded and the full query name is in the downloaded file's name

For other special characters: `@$%&+?\=;,"`:
- Save a Query containing one or more of the characters
- [ ] Ensure that Query export files can still be downloaded, the entire query name is present in the file name (special characters are either present in the download file name or replaced with `_`)
